src/_data/starters/11ty-starter-moody-person.json

```diff
@@ -0,0 +1,7 @@
+{
+	"url": "https://github.com/moody-person/11ty-starter",
+	"name": "11ty-starter",
+	"description": "Eleventy starter, using sass, nunjucks, rollup",
+	"author": "moody-person",
```

Comment:
@moody-person I think this is supposed to be a Twitter handle (not a GitHub/GitLab/etc account).

Reply:
ok, removed it.